readme.md

```diff
@@ -1,58 +1,119 @@
-# Pawn
+# Pawn Community Compiler
 
-[![Build Status][build_status]][build] [![Build Status - Windows][build_status_win]][build_win]
+[![Build Status][build_status]][build]
+[![Build Status - Windows][build_status_win]][build_win]
+
+## What
+
+This is a modified copy of the Pawn compiler version 3.2.3664 by Compuphase that
+fixes some bugs and adds a few features.
+
+This project was originally maintained and managed by Zeex who left the project
+due to lack of time on 2017-12-31. Thank you for all your hard work on keeping
+this project alive Zeex!
+
+The project is now maintained by the
+[Compiler Team](https://github.com/orgs/pawn-lang/teams/compiler) at the
+pawn-lang GitHub org.
 
 [Original readme](readme_compuphase.txt)
 
-## What is this?
+## Why
 
-This is a modified copy of the Pawn compiler version 3.2.3664 by Compuphase that fixes some bugs and adds a few features.
+The reason this exists is to:
 
-This project was originally maintained and managed by Zeex who left the project due to lack of time on 2017-12-31. Thank you for all your hard work on keeping this project alive Zeex!
+- Fix known bugs with the original compiler
+- Provide a better development experience for the SA:MP community
 
-The project is now maintained by the [Compiler Team](https://github.com/orgs/pawn-lang/teams/compiler) here at the pawn-lang GitHub org.
+You shouldn't have any reason _not_ to use this compiler. If you find problem,
```

Comment:
I think this actually bloats it with not-so-useful information. A link to the list of features is sufficient, IMO.